src/server/controllers/tasks.js

```diff
@@ -52,4 +52,27 @@ export class TaskManager {
       }
     });
   }
+  /**
+         *
+         *Updating a task
+         * @memberof TaskManager
+         */
+  updateTask() {
+    Task.findById(this.req.params.id).exec((error, tasks) => {
```

Comment:
This will return a single task... so have it singularly named.

Reply:
Alryt will change that

Reply:
have finished this

Reply:
Im done changing this

Reply:
And yet the property is still named `tasks`.